docs/querying/api.md

```diff
@@ -433,13 +433,35 @@ URL query parameters:
   series from which to read the label values. Optional.
 - `limit=<number>`: Maximum number of returned series. Optional. 0 means disabled.
 
-
 The `data` section of the JSON response is a list of string label values.
 
-This example queries for all label values for the `job` label:
+This example queries for all label values for the `job_name` label:
```

Comment:
Sorry for nitpicking again, but let's not use a "weird" label name in the first example. `job_name` is a weird because it's so common to have a `job` label. Why would somebody have `job_name`? If your intention is to have a label name with an underscore here, let' use one that is actually common like `http_status_code` (which could then be `http.status_code` in the example below to even allude to the different meaning of `.` and `_` in OTel).